src/main/org/apache/tools/ant/taskdefs/optional/jlink/ClassNameReader.java

```diff
@@ -86,14 +87,9 @@
                 values[i] = data.readUnsignedShort();
                 break;
 
-            case FIELDREF :
-            case METHODREF :
-            case INTERFACEMETHODREF :
-            case NAMEANDTYPE :
-                values[i] = data.readInt();
-                break;
+            case UNUSED :
```

Comment:
let me note a pattern here, Personally I do not like cases that fall through to the `default` case.